library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Spike-or-sharp-and-wave <nowiki> {suggestedTag=RPP-sharpness, suggestedTag=Number-of-RPP-phases, suggestedTag=RPP-triphasic-morphology, suggestedTag=RPP-absolute-amplitude, suggestedTag=RPP-relative-amplitude, suggestedTag=RPP-polarity, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Multifocal-finding, suggestedTag=Finding-frequency, suggestedTag=RPP-time-related-features} [Spike-and-wave or Sharp-and-wave (SW) - polyspike, spike or sharp wave consistently followed by a slow wave in a regularly repeating and alternating pattern (spike-wave-spike-wave-spike-wave), with a consistent relationship between the spike (or polyspike or sharp wave) component and the slow wave; and with no interval between one spike-wave complex and the next (if there is an interval, this would qualify as PDs, where each discharge is a spike-and- wave). (Source: Hirsch ea 2013; Suggested tags from Beniczky ea 2017, Table 8.)]</nowiki>
+
+
+'''Episode''' <nowiki> [Clinical episode or electrographic seizure. (Source: Beniczky ea 2013, Appendix S1.)]</nowiki>
+
+    * Epileptic-seizure <nowiki> {suggestedTag=Episode-consciousness-affected, suggestedTag=Episode-awareness, suggestedTag=Episode-prodrome, suggestedTag=Episode-tongue-biting} [The ILAE seizure classification divides seizures into focal, generalized onset, or unknown onset. (Source: Beniczky ea 2017, Table 9, Supplement 1; Selection-tree and list of seizure-types, according to the current ILAE seizure classification, Fisher ea 2017.)]</nowiki>
+        ** Focal-onset-epileptic-seizure <nowiki> {suggestedTag=Automatism-seizure, suggestedTag=Atonic-seizure, suggestedTag=Clonic-seizure, suggestedTag=Epileptic-spasm, suggestedTag=Hyperkinetic-seizure, suggestedTag=Myoclonic-seizure, suggestedTag=Tonic-seizure, suggestedTag=Autonomic-seizure, suggestedTag=Behavior-arrest-seizure, suggestedTag=Cognitive-seizure, suggestedTag=Emotional-seizure, suggestedTag=Sensory-seizure} [A focal seizure originates within networks limited to one hemisphere. They may be discretely localized or more widely distributed. Focal seizures may originate in subcortical structures. Focal seizures are optionally subdivided into focal aware and focal impaired awareness seizures. Specific motor and nonmotor classifiers may be added. (Source: Fisher ea 2017, Table 2 and Key Points; Suggested tags from Fisher ea 2017, Figure 2; Beniczky ea 2017, Supplement 1, ILAE seizure classification code I.)]</nowiki>
```

Comment:
Suggest:
They may be discretely -->  Focal seizures may be discretely

Reply:
This group might be a candidate for orthogonal representation --- with awareness part as a suggested tag?

Reply:
These seizures are specific subtypes of the Focal-onset-epileptic-seizure, with ILAE seizure classification codes I.A, I.B, I.C and I.D, so rather than being an additional tag, these are different seizure subtypes, which will favor maintaining the hierarchy.